tests/Dflydev/FigCookies/SetCookiesTest.php

```diff
@@ -20,13 +21,13 @@ class SetCookiesTest extends TestCase
      * @test
      * @dataProvider provideSetCookieStringsAndExpectedSetCookiesData
      */
-    public function it_creates_from_response(array $setCookieStrings, array $expectedSetCookies) : void
+    public function it_creates_from_response(array $setCookieStrings, array $expectedSetCookies): void
     {
-        /** @var ResponseInterface|ObjectProphecy $response */
-        $response = $this->prophesize(static::INTERFACE_PSR_HTTP_MESSAGE_RESPONSE);
-        $response->getHeader(SetCookies::SET_COOKIE_HEADER)->willReturn($setCookieStrings);
+        /** @var ResponseInterface|MockObject $response */
```

Comment:
prophecy is still usable, by loading a trait

Reply:
i have no idea if @simensen wants to keep prophecy

Reply:
Happy to drop it if it's easier at this point.